### ISSUE TITLE
Zoom Addition

### ISSUE THREAD

**Issue by Limonlol** [icerockdev/moko-maps#3]
Need:  Current map zoom, set max and min zoom

**Comment by Alex009** [comment]
will be available in 0.2.0 via:
```kotlin
suspend fun getCurrentZoom(): Float
suspend fun setCurrentZoom(zoom: Float)

suspend fun getZoomConfig(): ZoomConfig
suspend fun setZoomConfig(config: ZoomConfig)

data class ZoomConfig(
    val min: Float? = null,
    val max: Float? = null
)
```

**Comment by Alex009** [comment]
released in 0.2.0
